modules/auxiliary/gather/http/cve_2021_27850_apache_tapestry_hmac_key.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache Tapestry HMAC secret key leak',
+        'Description' => %q{
+          Find the HMAC secret key used in Java serizalization
```

Comment:
Could you add more details about the exploit here? How does this exploit access / retrieve the secret key?